### DISCUSSION TITLE
README.md에 무슨 내용을 넣으면 좋을까요 ?

### DISCUSSION THREAD

**Discussion by wisdomin121** [wanted-pre-onboarding-team-12th-7/pre-onboarding-12th-1-7#15] [Ideas]
README.md를 작성해야 하는 데 어떤 내용이 들어가면 좋을지 의견을 나눠보고 싶어요 !

---

![image](https://github.com/wanted-pre-onboarding-team-12th-7/pre-onboarding-12th-1-7/assets/46989954/01e4218e-a01a-4f93-ac39-5ca8e42a5e1f)

---

위의 사진은 기존 과제에서 요구했던 README.md 내용들이라 들어가는 게 맞다고 생각하고,
위의 요구사항 말고 어떤 내용을 넣으면 좋을까요 ?!

**Comment by SeungrokYoon** [comment]
지원자의 성명은 => 팀 멤버 소개란으로 변경하면 좋을 것 같습니다!

**Comment by salmontaker** [comment]
개발기간과 개발환경을 적는것도 좋을 것 같습니다!

- 개발기간

2023.08.22 ~ 2023.08.25

- 개발환경

[Node.js v18 (LTS)](https://nodejs.org)
[create-react-app](https://github.com/facebook/create-react-app)
[axios](https://github.com/axios/axios)

이런식으로요!

**Comment by SeungrokYoon** [comment]
1. 프론트엔드 아키텍처 도표도 넣으면 좋을 것 같습니다. 
   - 페이지 구성
   - 인증 플로우차트
   - 핵심 컴포넌트 설계 다이어그램

**Comment by SeungrokYoon** [comment]
위키와, 우리들이 겪었던 문제점 및 해결 과정 스토리를 간단히 소개하면 좋을 듯 합니다!
